src/forms/selectInput.story.jsx

```diff
@@ -49,6 +49,19 @@ storiesOf('SelectInput', module)
 			error='You forgot to name your horse!'
 		/>)
 	)
+	.add('with disabled options', () =>
+		(<SelectInput
+			label='Select a name for your horse'
+			id='horsename'
+			name='horsename'
+			required
+			options={[
+				{ label: 'Geoffrey', value: 'geoffrey', disabled: true },
+				{ label: 'Doctor Horse, MD Junior', value: 'drhorse', disabled: false },
```

Comment:
Instead of saying `disabled: false`, I think we can just leave it off

Reply:
ok. I just wanted to make sure passing attribute as false didnt do anything weird. I'll take it out though.